src/main/java/strategy/impl/BalanceActivityHandler.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.ActivityHandler;
+
+public class BalanceActivityHandler implements ActivityHandler {
+    @Override
+    public int count(int balance, int amount) {
+        return amount;
+    }
+}
```

Comment:
```suggestion
public class BalanceActivityHandler implements ActivityHandler {
    @Override
    public void handle(FruitTransaction transaction) {
        FruitStorage.fruitStorage.put(transaction.getFruit(), transaction.getAmount());
    }
}
```
activity handler should change the storage